src/include/bootloader/grub/helps.rb

```diff
@@ -80,8 +80,12 @@ def initialize_bootloader_grub_helps(_include_target)
         ),
         "password"          => _(
           "<p><b>Protect Boot Loader with Password</b><br>\n" \
-            "Define the password that will be required to access the boot menu. YaST will only accept the password if you repeat\n" \
-            "it in <b>Retype Password</b>.</p>"
+            "Define the password that will be required to access the boot menu. \n" \
+            "YaST will only accept the password if you repeat\n" \
+            "it in <b>Retype Password</b>. If <b>Allow to boot locked default \n" \
+            "entry without password</b> is checked, then without password is \n " \
+            "possible to let bootloader autoboot default entry with its kernel \n" \
+            "parameters.</p>"
```

Comment:
and for grub2 it behave same way. @mvidner can you look at it and propose possibly new labels and help text for it, that make better sense? something like
```
label = "Protect entry modification only"
helptext = "<b> Protect entry modification only</b> protects by password only modification of entry like loading different kernel or modify kernel parameters."
```